Meadowlark-js/src/repository/ElasticsearchRepository.ts

```diff
@@ -96,63 +102,33 @@ async function performSqlQuery(client: Client, query: string): Promise<any> {
   });
 }
 
-/**
```

Comment:
Removed the EdOrg/Student security clutter from queries, rather than work around it.